client/tests/nft_test.rs

```diff
@@ -120,83 +113,44 @@ async fn test_get_nft_collection_content_uri() -> anyhow::Result<()> {
     ));
     let res = assert_ok!(contract.get_collection_data().await);
 
-    assert_eq!(
-        res.collection_content,
-        MetaDataContent::External {
-            uri: "https://nft.fragment.com/numbers.json".to_string()
-        }
-    );
-
-    let meta_loader = NftColletionMetaLoader::default()?;
-    let content_res = assert_ok!(meta_loader.load(&res.collection_content).await);
-    assert_eq!(
-        content_res.name.as_ref().unwrap(),
-        &String::from("Anonymous Telegram Numbers")
-    );
-    assert_eq!(
-        content_res.image.as_ref().unwrap(),
-        &String::from("https://nft.fragment.com/numbers.svg")
-    );
-    Ok(())
-}
-
-#[tokio::test]
-async fn test_get_nft_collection_content_arkenston() -> anyhow::Result<()> {
-    common::init_logging();
-    let client = common::new_mainnet_client().await;
-    let factory = assert_ok!(TonContractFactory::builder(&client).build().await);
-    let contract = factory.get_contract(&assert_ok!(
-        "EQCshJXbbcn7cvSkaM0Z8NyI-2pNCJC5RTGZB-cRF-Pax1lY".parse()
+    let expected_owner_address = assert_ok!(TonAddress::from_base64_url(
+        &"EQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAM9c".to_string()
     ));
-    let res = assert_ok!(contract.get_collection_data().await);
-    let meta_loader = assert_ok!(NftColletionMetaLoader::default());
-    let content_res = assert_ok!(meta_loader.load(&res.collection_content).await);
-    assert_eq!(content_res.name.unwrap(), "ARKENSTON NFT");
-    assert_eq!(
-        content_res.image.unwrap(),
-        "https://static.ston.fi/stake-nft/i1.jpg"
-    );
+    let expected_res = NftCollectionData {
+        next_item_index: -1,
+        owner_address: expected_owner_address,
+        collection_content: MetaDataContent::External {
+            uri: "https://nft.fragment.com/numbers.json".to_string(),
+        },
+    };
+    assert_eq!(res, expected_res);
     Ok(())
 }
 
 #[tokio::test]
-async fn test_get_nft_collection_content_some() -> anyhow::Result<()> {
+async fn test_get_nft_address_by_index_is_valid() -> anyhow::Result<()> {
     common::init_logging();
     let client = common::new_mainnet_client().await;
-    let factory = assert_ok!(TonContractFactory::builder(&client).build().await);
+    let factory = TonContractFactory::builder(&client).build().await?;
     let contract = factory.get_contract(&assert_ok!(
-        "EQCbOjwru5tBb2aaXZEHbiTCVIYQ6yDNAe8SSZkP4CozibHM".parse()
+        "EQB2iHQ9lmJ9zvYPauxN9hVOfHL3c_fuN5AyRq5Pm84UH6jC".parse()
```

Comment:
I wonder why the address of contract has been changed. 
 Looks like the order of tests also has been changed. We should avoid mixing up test order is possible.

Reply:
It looks like a random git diff. I checked, and this address is correct and the same as the address before the refactor. You can check this in test 'test_get_nft_address_by_index' (line: 43 of this file).